packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -444,14 +446,8 @@ class _SummaryCardState extends State<SummaryCard> {
           InkWell(
             borderRadius: const BorderRadius.only(topRight: ROUNDED_RADIUS),
             onTap: widget.isSettingClickable
-                ? () async => Navigator.push<void>(
-                      context,
-                      MaterialPageRoute<void>(
-                        builder: (BuildContext context) =>
-                            const UserPreferencesPage(
-                          type: PreferencePageType.FOOD,
-                        ),
-                      ),
+                ? () => AppNavigator.of(context).push(
+                      AppRoutes.PREFERENCES(PreferencePageType.FOOD),
```

Comment:
Just checking: is it possible for the user to say, from the browser, "Ok you can send me to the app but this time I prefer to stay on the website"?

Reply:
Unfortunately, no (both on Android and iOS)

Reply:
That means we'd better check if we have all the major features on the app, compared to the website on smartphone.
In order to avoid user frustration.

Reply:
We're at feature parity, but some (hard to implement vs ROI) stuff is missing (like revision history).
Simple solution: on the product page, we could add an optional button to open the classic webpage in a webview.

Reply:
> We're at feature parity, but some (hard to implement vs ROI) stuff is missing (like revision history). Simple solution: on the product page, we could add an optional button to open the classic webpage in a webview.

I guess a button at the end of the product page would do the trick. Assuming we can exit the app using a URL the app is supposed to catch ;)